sdcm/results_analyze/__init__.py

```diff
@@ -1455,7 +1462,7 @@ def check_regression(self, test_name, test_results, test_details):  # pylint: di
             test_details=test_details,
             screenshots=test_details.pop("screenshots"),
         )
-        subject = f"Performance Regression: {test_name} - {format_timestamp(test_details['start_time'])}"
+        subject = f"[perf_regression] {test_name} - {format_timestamp(test_details['start_time'])}"
```

Comment:
Changing the subject at this point will be more painful than keeping it.
If one wants to search the inbox, one will need to search both the old subject and the new one while the short version isn't that much shorter.

I suggest not changing the subject.